src/components/Toggle/Toggle.module.css

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.toggle {
+  display: grid;
+  width: 44px;
+  height: 24px;
+}
+
+.toggle > * {
+  grid-area: 1/1;
+  width: 44px;
+  height: 24px;
+  box-sizing: border-box;
+}
+
+.toggle > input[type="checkbox"] {
+  opacity: 0;
+  margin: 0;
+  cursor: pointer;
+}
+
+.toggle > input[type="checkbox"][disabled] {
+  cursor: not-allowed;
+}
+
+.toggle-ui {
+  pointer-events: none;
+  width: 44px;
```

Comment:
From @janogarcia 

> I'd suggest that sizing/dimensions are defined as rem instead of px, that way the components will adapt gracefully to the user's system font size, in a similar way to changing the font size at the system level in iOS/Android.
A few exceptions may apply, like fixed width borders (e.g., we want to keep a border at1px no matter the system font size)

Reply:
Done in this context, there are still a few places in the codebase that would benefit from this change.